.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,77 @@
+name: Performance benchmarks
+
+on:
+  pull_request_target:
+    types: [opened, ready_for_review]
+    branches:
+      - main
+  issue_comment:
+    types: [created]
+
+permissions:
```

Comment:
Are these needed with pull_request_target?

Reply:
I think these are the default values, but explicit might be better than implicit here, especially if they ever change the defaults.

Reply:
This does make me a little nervous with "write" permissions. [Did you try it with "none" or "read" ](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs)  or is it even necessary, looking through some other projects with `.github/workflows/benchmarks.yml` I did not see others with permissions but it was a cursory look and there are many variants of repo setups and workflows.

Reply:
> [Did you try it with "none" or "read" ](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs)or is it even necessary,

Yes, doesn't work. Basically the whole discussion of Friday: https://github.com/projectmesa/mesa/pull/1978#issuecomment-1901104588. This is because there is write permission needed to place a comment.